juniper_official/Solutions/Traffic-Blackhole/fabric-interrups.rule

```diff
@@ -0,0 +1,112 @@
+/*
+	* REQ: COL-1109
+	* Rule to collect Fabric Block CM Errors and Interrupts for Blackhole Detection RCA.
+*/
+healthbot {
+	topic hardware.fpc {
+		rule fabric-interrupts {
+			keys [ fpc-name error-name ];
+			synopsis "Fabric Block CM Errors and Interrupts";
+			description "Fabric Block CM Errors and Interrupts";
+			sensor sensor-interrupts {
+				synopsis "Fabric Block CM Errors and Interrupts";
+				description "Fabric Block CM Errors and Interrupts";
+				open-config {
+					sensor-name /components/component/integrated-circuit/pipeline-counters/errors/fabric-block/fabric-block-error/;
```

Comment:
Change this /components/component to avoid duplicate subscriptions and reduce data from devices

Reply:
Changed path to: /components/component/